src/docs/development/state-mgmt.md

```diff
@@ -0,0 +1,37 @@
+---
+title: State management
+description: How to structure an app to manage the state of the data flowing through it.
+---
```

Comment:
This file was added under `src/docs/development/state-mgmt.md`, but the URL path used in the sidenav is `permalink: /docs/development/data-and-backend/state-mgmt` (see https://github.com/flutter/website/pull/1614/files#diff-17a8fae72c56a55f7908c522f5afc95aR82), so this file will need to be moved.